pkg/engine/raw.go

```diff
@@ -107,52 +109,60 @@ func (r *Raw) Process(ctx context.Context, conn context.Context, PAT string, ske
 		return
 	}
 
-	r.initialRun = false
 }
 
 func (r *Raw) rawPodman(ctx, conn context.Context, path string, prev *string) error {
+	if prev != nil {
+		klog.Infof("Creating podman container from %s: %s", path, *prev)
+	} else {
+		klog.Infof("Creating podman container from %s", path)
+	}
 
-	klog.Infof("Creating podman container from %s", path)
+	var raw *RawPod
 
-	rawFile, err := ioutil.ReadFile(path)
-	if err != nil {
-		return err
-	}
+	if path != deleteFile {
+		rawFile, err := ioutil.ReadFile(path)
+		if err != nil {
+			return err
+		}
 
-	raw, err := rawPodFromBytes(rawFile)
-	if err != nil {
-		return err
-	}
+		klog.Info("Done creating")
 
-	klog.Infof("Identifying if image exists locally")
+		raw, err = rawPodFromBytes(rawFile)
+		if err != nil {
+			return err
+		}
 
-	err = detectOrFetchImage(conn, raw.Image, r.PullImage)
-	if err != nil {
-		return err
+		klog.Infof("Identifying if image exists locally")
+
+		err = detectOrFetchImage(conn, raw.Image, r.PullImage)
+		if err != nil {
+			return err
+		}
 	}
 
 	// Delete previous file's podxz
 	if prev != nil {
-		raw, err := rawPodFromBytes([]byte(*prev))
+		rawPrev, err := rawPodFromBytes([]byte(*prev))
 		if err != nil {
 			return err
 		}
 
-		err = deleteContainer(conn, raw.Name)
+		err = deleteContainer(conn, rawPrev.Name)
 		if err != nil {
```

Comment:
nit - see note below about single line (not w/ your code, but can single line in this PR)